### PR TITLE
i18n the bool repr

### PR DESCRIPTION
**Pull request by austin-schick**

### CODE REVIEW

File: www/src/py_int.js [hunk 1]
```diff
@@ -913,8 +913,14 @@ bool.__or__ = function(self, other){
...
 bool.__pos__ = $B.int_or_bool
 
+BOOL_STRINGS = {
+  "en": ["True", "False"],
+  "es": ["Verdadero", "Falso"],
+  "de": ["Wahr", "Falsch"],
+}
 bool.__repr__ = bool.__str__ = function(self){
-    return self ? "True" : "False"
+    let language = $B.language.slice(0, 2);
```
Comment (schmave) [inline www/src/py_int.js:922]: I think it would be clearer if this was `$B.cmuGraphicsLanguage`

File: www/src/py_int.js [hunk 2]
```diff
@@ -913,8 +913,14 @@ bool.__or__ = function(self, other){
...
+    return BOOL_STRINGS[language][self ? 0 : 1]
```
Comment (schmave) [inline www/src/py_int.js:923]: Fall back to English strings if the language is not found?

File: www/src/py_int.js (hunk 1 above)
Comment (austin-schick) [reply www/src/py_int.js:922]: That makes sense. `$B.language` already exists, but that isn't _really_ what we care about here

File: www/src/py_int.js (hunk 2 above)
Comment (austin-schick) [reply www/src/py_int.js:923]: I think that should be impossible, but it can't hurt :)

**Review by schmave — APPROVED** [review]

**Pull request merged**